packages/node_modules/@node-red/runtime/lib/nodes/credentials.js

```diff
@@ -239,7 +239,15 @@ var api = module.exports = {
                     throw error;
                 }
             } else {
-                credentialCache = credentials;
+                if (encryptionEnabled) {
+                    // Our config expects the credentials to be encrypted but the encrypted object is not found
+                    log.warn(log._("nodes.credentials.encryptednotfound"))
```

Comment:
Either kebab-case or camelCase - never alllowercase
```suggestion
                    log.warn(log._("nodes.credentials.encryptedNotFound"))
```